src/pages/Home/Home.tsx

```diff
@@ -1,15 +1,40 @@
 import { type FC } from 'react';
 
-import { Container } from './styles';
+import Progress from '../../components/ProgressBar';
+import CardQuest from '../../components/QuestCard';
 
+import { Container, ProgressContainer, QuestsWrapper } from './styles';
 interface HomeTypes {
   initial?: number;
 }
 
 const Home: FC<HomeTypes> = () => {
   return (
     <Container>
-      <h1>Home</h1>
+      <h1>É hora do jogo!</h1>
+      <h2>Complete as quests corretamente e obtenha uma surpresa no final</h2>
+
+      <ProgressContainer>
+        <Progress progress={0} progressTitle="Quests finalizadas" />
+
+        {/* TODO: Points badge here */}
+        <div> </div>
+      </ProgressContainer>
+
+      <QuestsWrapper>
+        <CardQuest
```

Comment:
Since you're working on the home page, I'd use the global context instead of mocking the values.

You can see an example of the syntax to consume the global storage on the code I removed on this commit's `src/App.tsx`:
https://github.com/KevBeltrao/conectados/commit/6dedb33ae72057b6065e56f4241473c8cb557539

PS: Add the logic to the `src/pages/Home/index.tsx` and pass the desired values as props.

(Remember to also pass the progress instead of letting `0`).